### ISSUE TITLE
[Support]: Frigante won't load after HASS Core update 9.3 to 9.4

### ISSUE THREAD

**Issue by jnrivra** [blakeblackshear/frigate#3852]
### Describe the problem you are having

Frigate won't load after HASS Core update 9.3 to 9.4

The camera used to work with that exact config.

Things I've tried:
- Mqtt is working and connecting properly
- Uninstalling frigate addon and integration 
- restored from past backup in home assistant
- the log is the debug version
- it also used to work with hard accel but not its not longer working also
ffmpeg:
  hwaccel_args:
    - -hwaccel
    - vaapi
    - -hwaccel_device
    - /dev/dri/renderD128
    - -hwaccel_output_format
    - yuv420p
    
    
 thank you so much! I've been hours and hours on this :/

### Version

3.1

### Frigate config file

```yaml
mqtt:
  host: 192.168.0.150
  user: mqtt
  password: mqttPassword
  
cameras:
  Hik1:
    ffmpeg:
      inputs:
        - path: rtsp://admin:admin123@192.168.0.27:554/h264/ch2/main/av_stream        
          roles:
            - rtmp


logger:
  default: debug
```

### Relevant log output

```shell
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[2022-09-16 09:26:31] frigate.app                    INFO    : Starting Frigate (0.10.1-83481af)
[2022-09-16 09:26:31] frigate.app                    INFO    : Creating directory: /tmp/cache
Starting migrations
[2022-09-16 09:26:31] peewee_migrate                 INFO    : Starting migrations
[2022-09-16 09:26:31] peewee                         DEBUG   : ('CREATE TABLE IF NOT EXISTS "migratehistory" ("id" INTEGER NOT NULL PRIMARY KEY, "name" VARCHAR(255) NOT NULL, "migrated_at" DATETIME NOT NULL)', [])
[2022-09-16 09:26:31] peewee                         DEBUG   : ('SELECT "t1"."id", "t1"."name", "t1"."migrated_at" FROM "migratehistory" AS "t1" ORDER BY "t1"."id"', [])
There is nothing to migrate
[2022-09-16 09:26:31] peewee_migrate                 INFO    : There is nothing to migrate
[2022-09-16 09:26:31] detector.cpu                   INFO    : Starting detection process: 221
[2022-09-16 09:26:31] frigate.app                    INFO    : Output process started: 222
[2022-09-16 09:26:31] ws4py                          INFO    : Using epoll
[2022-09-16 09:26:31] frigate.app                    INFO    : Camera processor started for Hik1: 226
[2022-09-16 09:26:31] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-09-16 09:26:31] frigate.app                    INFO    : Capture process started for Hik1: 229
[2022-09-16 09:26:31] ws4py                          INFO    : Using epoll
[2022-09-16 09:26:31] peewee.sqliteq                 DEBUG   : received query UPDATE "event" SET "end_time" = ("event"."start_time" + ?) WHERE ("event"."end_time" IS ?)
[2022-09-16 09:26:31] peewee                         DEBUG   : ('UPDATE "event" SET "end_time" = ("event"."start_time" + ?) WHERE ("event"."end_time" IS ?)', [30, None])
[2022-09-16 09:27:31] frigate.record                 DEBUG   : Start expire recordings (new).
[2022-09-16 09:27:31] frigate.record                 DEBUG   : Start deleted cameras.
[2022-09-16 09:27:31] peewee                         DEBUG   : ('SELECT "t1"."id", "t1"."camera", "t1"."path", "t1"."start_time", "t1"."end_time", "t1"."duration", "t1"."motion", "t1"."objects" FROM "recordings" AS "t1" WHERE (("t1"."camera" NOT IN (?)) AND ("t1"."end_time" < ?))', ['Hik1', 1663331251.702811])
[2022-09-16 09:27:31] frigate.record                 DEBUG   : Expiring 0 recordings
[2022-09-16 09:27:31] peewee.sqliteq                 DEBUG   : received query DELETE FROM "recordings" WHERE (0 = 1)
[2022-09-16 09:27:31] peewee                         DEBUG   : ('DELETE FROM "recordings" WHERE (0 = 1)', [])
[2022-09-16 09:27:31] frigate.record                 DEBUG   : End deleted cameras.
[2022-09-16 09:27:31] frigate.record                 DEBUG   : Start all cameras.
[2022-09-16 09:27:31] frigate.record                 DEBUG   : Start camera: Hik1.
[2022-09-16 09:27:31] peewee                         DEBUG   : ('SELECT "t1"."id", "t1"."camera", "t1"."path", "t1"."start_time", "t1"."end_time", "t1"."duration", "t1"."motion", "t1"."objects" FROM "recordings" AS "t1" WHERE (("t1"."camera" = ?) AND ("t1"."end_time" < ?)) ORDER BY "t1"."start_time"', ['Hik1', 1663330951.704347])
[2022-09-16 09:27:31] frigate.record                 DEBUG   : Expiring 0 recordings
[2022-09-16 09:27:31] peewee.sqliteq                 DEBUG   : received query DELETE FROM "recordings" WHERE (0 = 1)
[2022-09-16 09:27:31] peewee                         DEBUG   : ('DELETE FROM "recordings" WHERE (0 = 1)', [])
[2022-09-16 09:27:31] frigate.record                 DEBUG   : End camera: Hik1.
[2022-09-16 09:27:31] frigate.record                 DEBUG   : End all cameras.
[2022-09-16 09:27:31] frigate.record                 DEBUG   : End expire recordings (new).
[2022-09-16 09:27:31] frigate.record                 DEBUG   : Start expire files (legacy).
[2022-09-16 09:27:31] peewee                         DEBUG   : ('SELECT "t1"."id", "t1"."camera", "t1"."path", "t1"."start_time", "t1"."end_time", "t1"."duration", "t1"."motion", "t1"."objects" FROM "recordings" AS "t1" ORDER BY "t1"."start_time" LIMIT ? OFFSET ?', [1, 0])
[2022-09-16 09:27:31] frigate.record                 DEBUG   : Oldest recording in the db: 1663331251.706057
[2022-09-16 09:27:31] frigate.record                 DEBUG   : End expire files (legacy).
[2022-09-16 09:31:31] peewee                         DEBUG   : ('SELECT DISTINCT "t1"."label" FROM "event" AS "t1" WHERE ("t1"."camera" NOT IN (?))', ['Hik1'])
[2022-09-16 09:31:31] peewee                         DEBUG   : ('SELECT DISTINCT "t1"."label" FROM "event" AS "t1" WHERE ("t1"."camera" = ?)', ['Hik1'])
[2022-09-16 09:31:31] peewee                         DEBUG   : ('SELECT DISTINCT "t1"."label" FROM "event" AS "t1" WHERE ("t1"."camera" NOT IN (?))', ['Hik1'])
[2022-09-16 09:31:31] peewee                         DEBUG   : ('SELECT DISTINCT "t1"."label" FROM "event" AS "t1" WHERE ("t1"."camera" = ?)', ['Hik1'])
[2022-09-16 09:31:31] peewee.sqliteq                 DEBUG   : received query with grouped_events as (
          select id,
            label,
            camera,
          	has_snapshot,
          	has_clip,
          	row_number() over (
              partition by label, camera, round(start_time/5,0)*5
              order by end_time-start_time desc
            ) as copy_number
          from event
        )
        select distinct id, camera, has_snapshot, has_clip from grouped_events
        where copy_number > 1;
[2022-09-16 09:31:31] peewee                         DEBUG   : ('with grouped_events as (\n          select id,\n            label,\n            camera,\n          \thas_snapshot,\n          \thas_clip,\n          \trow_number() over (\n              partition by label, camera, round(start_time/5,0)*5\n              order by end_time-start_time desc\n            ) as copy_number\n          from event\n        )\n\n        select distinct id, camera, has_snapshot, has_clip from grouped_events\n        where copy_number > 1;', [])
[2022-09-16 09:31:31] peewee.sqliteq                 DEBUG   : received query DELETE FROM "event" WHERE (0 = 1)
[2022-09-16 09:31:31] peewee                         DEBUG   : ('DELETE FROM "event" WHERE (0 = 1)', [])
[2022-09-16 09:31:31] peewee.sqliteq                 DEBUG   : received query DELETE FROM "event" WHERE (("event"."has_clip" = ?) AND ("event"."has_snapshot" = ?))
[2022-09-16 09:31:31] peewee                         DEBUG   : ('DELETE FROM "event" WHERE (("event"."has_clip" = ?) AND ("event"."has_snapshot" = ?))', [False, False])
[2022-09-16 09:36:31] peewee                         DEBUG   : ('SELECT DISTINCT "t1"."label" FROM "event" AS "t1" WHERE ("t1"."camera" NOT IN (?))', ['Hik1'])
[2022-09-16 09:36:31] peewee                         DEBUG   : ('SELECT DISTINCT "t1"."label" FROM "event" AS "t1" WHERE ("t1"."camera" = ?)', ['Hik1'])
[2022-09-16 09:36:31] peewee                         DEBUG   : ('SELECT DISTINCT "t1"."label" FROM "event" AS "t1" WHERE ("t1"."camera" NOT IN (?))', ['Hik1'])
[2022-09-16 09:36:31] peewee                         DEBUG   : ('SELECT DISTINCT "t1"."label" FROM "event" AS "t1" WHERE ("t1"."camera" = ?)', ['Hik1'])
[2022-09-16 09:36:31] peewee.sqliteq                 DEBUG   : received query with grouped_events as (
          select id,
            label,
            camera,
          	has_snapshot,
          	has_clip,
          	row_number() over (
              partition by label, camera, round(start_time/5,0)*5
              order by end_time-start_time desc
            ) as copy_number
          from event
        )
        select distinct id, camera, has_snapshot, has_clip from grouped_events
        where copy_number > 1;
[2022-09-16 09:36:31] peewee                         DEBUG   : ('with grouped_events as (\n          select id,\n            label,\n            camera,\n          \thas_snapshot,\n          \thas_clip,\n          \trow_number() over (\n              partition by label, camera, round(start_time/5,0)*5\n              order by end_time-start_time desc\n            ) as copy_number\n          from event\n        )\n\n        select distinct id, camera, has_snapshot, has_clip from grouped_events\n        where copy_number > 1;', [])
[2022-09-16 09:36:31] peewee.sqliteq                 DEBUG   : received query DELETE FROM "event" WHERE (0 = 1)
[2022-09-16 09:36:31] peewee                         DEBUG   : ('DELETE FROM "event" WHERE (0 = 1)', [])
[2022-09-16 09:36:31] peewee.sqliteq                 DEBUG   : received query DELETE FROM "event" WHERE (("event"."has_clip" = ?) AND ("event"."has_snapshot" = ?))
[2022-09-16 09:36:31] peewee                         DEBUG   : ('DELETE FROM "event" WHERE (("event"."has_clip" = ?) AND ("event"."has_snapshot" = ?))', [False, False])
```

### FFprobe output from your camera

```shell
sorry, couldn't figure out how to get this
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

hikvision

### Any other information that may be helpful

this is the log with only INFO

[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[2022-09-16 09:50:38] frigate.app                    INFO    : Starting Frigate (0.10.1-83481af)
[2022-09-16 09:50:38] frigate.app                    INFO    : Creating directory: /tmp/cache
Starting migrations
[2022-09-16 09:50:38] peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
[2022-09-16 09:50:38] peewee_migrate                 INFO    : There is nothing to migrate
[2022-09-16 09:50:38] frigate.app                    INFO    : Output process started: 222
[2022-09-16 09:50:38] frigate.app                    INFO    : Camera processor started for Hik1: 225
[2022-09-16 09:50:38] frigate.app                    INFO    : Capture process started for Hik1: 227

**Comment by NickM-27** [comment]
A few things here:
1. A core update wouldn't affect frigate (or any addon) as they're entirely isolated, it must've been an OS update.
2. Please turn off debug logging, the default logging is already helpful enough to know why frigate is stopping and the debug logging just creates a lot of clutter.
3. You say frigate won't load but the logs show it is loading just fine, so what is actually not working?

**Comment by jnrivra** [comment]
thanks for the fast response!

1. I thought the same, a couple days ago I changed the cameras names but even after multiple restarts, nothing went wrong, only after the update. 

2. this are the logs without debug.

[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[2022-09-16 10:09:43] frigate.app                    INFO    : Starting Frigate (0.10.1-83481af)
[2022-09-16 10:09:43] frigate.app                    INFO    : Creating directory: /tmp/cache
Starting migrations
[2022-09-16 10:09:43] peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
[2022-09-16 10:09:43] peewee_migrate                 INFO    : There is nothing to migrate
[2022-09-16 10:09:43] detector.cpu                   INFO    : Starting detection process: 221
[2022-09-16 10:09:43] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-09-16 10:09:43] frigate.app                    INFO    : Output process started: 223
[2022-09-16 10:09:43] frigate.app                    INFO    : Camera processor started for Hik1: 227
[2022-09-16 10:09:43] ws4py                          INFO    : Using epoll
[2022-09-16 10:09:43] frigate.app                    INFO    : Capture process started for Hik1: 229
[2022-09-16 10:09:43] ws4py                          INFO    : Using epoll

3. When I press start, the app go to running state and I try to open the frigate app, it promps: "Add-on is not running. Please start it first"

It's weird, If i debug with the hardware accel in the config, I can actually see the mqtt from the camera going out . but Hass won't see Frigate as ON

**Comment by NickM-27** [comment]
So you start the frigate addon, does it then stop by itself and these are all the logs you see?

**Comment by jnrivra** [comment]
maybe there's a way to completely uninstall everything related to frigate, if you can help me with that I could try it

**Comment by NickM-27** [comment]
Okay, if after it stops those are the only logs there then we usually see this because there is an issue with MQTT. It can see your MQTT server but it is unable to communicate with it. https://github.com/blakeblackshear/frigate/issues/3399 is an example of this

**Comment by jnrivra** [comment]
That's interesting, I'll try to reinstall MQTT and Frigate.

**Comment by jnrivra** [comment]
Manage to solve it. Thank you a lot Nick!! I did all this:

- Stop and Start MQTT
- reinstall frigate addon and integration
- Create new names for all the cameras in frigate.yml (it seems there was an issue with that also)

So here comes the weird part. Nick you were right, the addon is working properly but the UI in HASS shows only the "start" button and no data of hostname, cpu or ram ussage. It looks that is not working but, when after clicking start, I added the integration, new names appeared and bam. everything is working.

Still the UI shows as if Frigate is stopped, but anyway. works :D

thanks again!
